src/components/NewCommentForm.tsx

```diff
@@ -75,25 +181,45 @@
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', {
+              'is-danger': false,
```

Comment:
The 'is-danger' class is always set to false. Consider updating this logic to reflect the actual error state for the body field.